build.sbt

```diff
@@ -215,16 +216,15 @@ lazy val packagingSettings = Seq(
   maintainer := "Mesosphere Package Builder <support@mesosphere.io>",
   serverLoading := None, // We override this to build for each supported system loader in the packageLinux alias
   debianPackageDependencies in Debian := Seq("java8-runtime-headless", "lsb-release", "unzip", s"mesos (>= ${Dependency.V.MesosDebian})"),
+  packageArchitecture in Debian := "amd64",
+  version in Debian := s"${version.value}-1.ubuntu1804",
+
+  packageArchitecture in Rpm := "x86_64",
```

Comment:
Is this correct? I don't believe we ship any x86_64 code in the package. It should be `arch`.

Reply:
@lloesche suggested this. He can explain.

Reply:
@timcharper see my comment in the corresponding Jira. The repo server was originally only for Mesos and we only had a x86_64 architecture. Our instructions originally told folks to only add this repo to their yum config. We never changed that for Marathon, which is why the original marathon tagged it's packages as x86_64 even though noarch would be correct. We later created a mesosphere-el-repo-7-3.noarch.rpm package which installs existing archs and repos for the user and even updated our blog post to have the correct instructions but by that time a lot of users were already using the repo and most likely never updated it. So we deemed it safer to continue using the x86_64 arch in the packages metadata instead of having users stop receiving updates.